### PR TITLE
#168049993 Setup API documentation using swagger

### PR DESCRIPTION
**Pull request by amilykassim**
#### What does this PR do?
- Setup API documentation
#### Description of Task to be completed?
- Adding a swagger package called swagger-ui-express
- Creating a swagger.json file
- Rendering swagger.json file when hitting `/api-docs` endpoint.
#### How should this be manually tested?
- Start your server by running `npm run dev` script
- Call this endpoint on your browser: `localhost:3000/api-docs`
- Then boom, swagger is already presented on your screen like magic
#### What are the relevant pivotal tracker stories?
[#168049993](https://www.pivotaltracker.com/story/show/168049993)
#### What are the relevant Github Issues?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

**Review by gik-hub — APPROVED** [review]
Okay

**Review by mucyomiller — APPROVED** [review]
LGTM

**Review by didasmbalanya — APPROVED** [review]

**Review by Jordybastien — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "dev": "NODE_ENV=development nodemon --exec babel-node src/index.js",
```
Comment (Paulvitalis200) [inline package.json:7]: Could you install `nodemon` too. I can't see it in the `package.json` file.

Comment (amilykassim) [reply package.json:7]: Sure, let me do that!

**Review by Paulvitalis200 — APPROVED** [review]

**Pull request merged**